proposals/dao/fip_cvx.ts

```diff
@@ -0,0 +1,59 @@
+import { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import {
+  DeployUpgradeFunc,
+  NamedContracts,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { getImpersonatedSigner } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+
+chai.use(CBN(ethers.BigNumber));
+
+const cErc20PluginImpl = '0xb9bEECD1A582768711dE1EE7B0A1d582D9d72a6C';
+const plugin = '0x2a810409872afc346f9b5b26571fd6ec42ea4849';
```

Comment:
these should be in mainnetAddresses.ts, not defined here

Reply:
also I couldn't find these addresses on mainnet

Reply:
he probably deployed on a local fork because Rari contracts have to be deployed too, for it to work, and they're in another repo

Reply:
Yeah need to wait until the stuff is audited to deploy